addons/sourcemod/scripting/shavit-misc.sp

```diff
@@ -1969,13 +1969,18 @@ public void Shavit_OnRestart(int client)
 			}
 		}
 
-		CreateTimer(0.1, Respawn, GetClientSerial(client), TIMER_FLAG_NO_MAPCHANGE);
+		DataPack pack;
+		CreateDataTimer(0.1, Respawn, pack, TIMER_FLAG_NO_MAPCHANGE);
+		pack.WriteCell(GetClientSerial(client));
+		pack.WriteCell(track);
 	}
 }
 
-public Action Respawn(Handle Timer, any data)
+public Action Respawn(Handle Timer, DataPack pack)
 {
-	int client = GetClientFromSerial(data);
+	pack.Reset();
+	int client = GetClientFromSerial(pack.ReadCell());
+	int track = pack.ReadCell();
```

Comment:
needs a `delete pack;` here

Reply:
My fault, sorry! The only mismatch with code styling here is that everywhere else in the code, I initialize variable with their default values. For example, `Handle` (such as `DataPack`) would be initialized as `= null`, even if it changes nothing, I'd like to keep the code style consistent.

Went through it again and everything else seems completely fine!